src/pug/blog/report_17.pug

```diff
@@ -0,0 +1,53 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #17
+
+block content
+  h2 Progress report #17
+  +news_header('6 October 2023', 'report_16', '')
+
+  h3 Ctrl app
+  p We released firmware #[+ax(org.fw_release+'0.92.0') 0.92.0] along the new #[+ax('https://ctrl.inputlabs.io') Ctrl app] for the Alpakka controller. We decided that having a minimalistic app was the best solution to address the feedback from the community, and the desire to have a simpler way to modify profiles. The app is aligned with Input Labs principles, so it is completely #[em optional], #[em open-source], and #[em multi-platform].
+  p Some of the features the Ctrl app have already:
+  ul
+    li Changing controller configuration (mouse sensitivity, touch responsiveness, deadzone, protocol).
+    li Functions such as calibration and factory reset.
+    li Simple access to the firmware logs, for troubleshooting any issues while DIY'ing the controller.
+
+  p Features planned for later:
+  ul
+    li Profile edit.
+    li Offline mode (based on PWA).
+    li Input tester.
+
+  p The app is accessible at #[+ax('https://ctrl.inputlabs.io') ctrl.inputlabs.io] and it is compatible with #[em Windows], #[em Linux], and the #[em Steam Deck].
+
+  img.wm.mr.mt(src='/static/manual/ctrl_app_screenshot_1.png')
+  img.wm.mr.mt(src='/static/manual/ctrl_app_screenshot_2.png')
+  hr
+
+  h3 Research with Pico-W Bluetooth continues
+  p We are actively testing different approaches to use the Raspberry Pi Pico-W Bluetooth, with the focus on bringing the latency to the minimum.
+  p As you can imagine the Bluetooth standard is quite complex nowadays, but the good folks from #[+ax('https://bluekitchen-gmbh.com') Blue Kitchen] (creators of the Bluetooth library the Pico is using) are helping us figuring out all the small details, and we deeply appreciate their support.
```

Comment:
Thanks to the help from the good folk of BK (creators, ...) we were able to fix bugs and find ways to reduce the latency.